griff-m001/Data-Collection-Project
# Added Express.js, Materialize CSS, and organized static files in 'public' directory.

###**Description**
This pull request includes the following changes:

Added Express.js framework to the project for server-side functionality.
Added the Materialize CSS package for better styling and design.
Modified a section of the code to improve performance and readability.
Moved the static files to the 'public' directory for better organization and accessibility.
Motivation and Context
The addition of Express.js and Materialize CSS to the project will enhance the functionality and user experience. The modified code section will improve performance, and moving static files to the 'public' directory will make them easily accessible for the server.

##**How to Test**
To test this change, clone the repository and run the project. Verify that the Express.js framework and Materialize CSS package are installed and functioning correctly. Also, ensure that the modified code section is working as expected, and the static files are being served from the 'public' directory.

##**Checklist**
 I have tested the changes and ensured they are working as expected.
 I have updated the documentation to reflect the changes.
 I have added appropriate comments to the code.
 I have followed the project's coding standards and guidelines.
